scripts/variantstore/wdl/GvsCreateFilterSet.wdl

```diff
@@ -43,7 +43,7 @@ workflow GvsCreateFilterSet {
   String fq_filter_sites_destination_table =    "~{project_id}.~{dataset_name}.filter_set_sites"
   String fq_filter_set_info_destination_table = "~{project_id}.~{dataset_name}.filter_set_info"
 
-  String filter_set_info_destination_table_schema = "filter_set_name:string,type:string,location:integer,ref:string,alt:string,calibration_sensitivity:float,score:float,vqslod:float,culprit:string,training_label:string,yng_status:string"
+  String filter_set_info_destination_table_schema = "filter_set_name:string,type:string,location:integer,ref:string,alt:string,calibration_sensitivity:float,score:float,score2:float,vqslod:float,culprit:string,training_label:string,yng_status:string"
```

Comment:
what's this change for?

Reply:
That's just there for testing. Gone in 10 seconds.